client_cli/src/main.rs

```diff
@@ -104,6 +104,8 @@ pub enum Subcommand {
     Events(events::Args),
     /// The subcommand related to Wasm
     Wasm(wasm::Args),
+    /// The subcommand related to block streaming
```

Comment:
note to self: update bash guide here https://hyperledger.github.io/iroha-2-docs/guide/bash.html#_2-configuring-iroha-2